mcs/class/corlib/System.Threading/Thread.cs

```diff
@@ -90,7 +90,6 @@ sealed class InternalThread : CriticalFinalizerObject {
 		private IntPtr interrupt_on_stop;
 		private IntPtr flags;
 		private IntPtr thread_pinning_ref;
-		private IntPtr async_invoke_method;
```

Comment:
This is a breaking change in the BCL ABI, it must come with a bump in the corlib version.